content/06.model-complexity.md

```diff
@@ -17,25 +16,25 @@ Additional approaches like combining random forests with resampling without repl
 Regularization is an approach by which a penalty is added to the model to avoid making large prediction errors. 
-Some examples of regularization approaches include ridge regression, LASSO regression, and Elastic-net regression, among others.
-Regularization is often used in rare variant discovery and immune cell signature discovery studies which are useful examples that also need to accommodate sparsity of data like rare diseases.
-Among various regularization methods hybrid applications of LASSO for capturing combinations of rare and common variants associated with specific traits have proven beneficial [@doi:10.1186/1753-6561-5-s9-s113].
+Some examples of ML methods with regularization include ridge regression, LASSO regression, and elastic net regression, among others.
+Regularization is often used in rare variant discovery and immune cell signature discovery studies; much like rare disease, these examples need to accommodate sparsity in data.
+For example, LASSO has been used to capturing combinations of rare and common variants associated with specific traits have proven beneficial [@doi:10.1186/1753-6561-5-s9-s113].
 In this example, applying LASSO regularization reduced the number of common variants included as features in the final analysis generating a simpler model while reducing error in the association of common and rare variants with a specific trait. 
 In the context of rare immune cell signature discovery, variations of elastic-net regression were found to outperform other regression approaches [@doi:10.1111/j.1467-9868.2005.00503.x; @doi:10.1016/j.compbiomed.2015.10.008; @doi:10.1186/s12859-019-2994-z]. 
-Thus regularization methods like LASSO or elastic-net have been methods of choice while working with rare observations.[@doi:10.1371/journal.pgen.1004754] 
-Furthermore, in applications of neural network based methods on data from rare diseases like acute myeloid luekemia (AML) or tuberous sclerosis (TS), other methods of regularization like Kullback–Leibler divergence (KL divergence) and dropout have been used respectively.
+Thus, regularization methods like LASSO or elastic-net are beneficial in ML with rare observations, and are worth exploring in the context of rare diseases.[@doi:10.1371/journal.pgen.1004754] 
+Other examples of regularization that have been successfully applied to rare disease ML include Kullback–Leibler divergence (KL divergence) and dropout regularization.
 In a study using a variational auto encoder (VAE) for dimensionality reduction in gene expression data from AML samples, the KL divergence between the input data and its low dimensional representation provided the regularizing penalty for the model.[@doi:10.1101/278739; @doi:10.48550/arXiv.1312.6114]
```

Comment:
Need to define AML now that the sentence before this one no longer does.

Reply:
```suggestion
In a study using a variational auto encoder (VAE) for dimensionality reduction in gene expression data from acute myeloid leukemia (AML) samples, the KL divergence between the input data and its low dimensional representation provided the regularizing penalty for the model.[@doi:10.1101/278739; @doi:10.48550/arXiv.1312.6114]
```